cabal.project

```diff
@@ -1,6 +1,8 @@
 packages:
  .
 
+shared: True
```

Comment:
This is needed, otherwise we get errors like 

```
[1 of 8] Compiling Text.URI.Parser.Text.Utils ( Text/URI/Parser/Text/Utils.hs, dist/build/Text/URI/Parser/Text/Utils.o, dist/build/Text/URI/Parser/Text/Utils.dyn_o )
Text/URI/Parser/Text/Utils.hs:34:1: error: [GHC-47808]
    Failed to load dynamic interface file for Text.Megaparsec:
      Exception when reading interface file  /home/hasufell/.cabal/store/ghc-9.10.1.20241021/megaparsec-9.6.1-e7391810ad72bf7d47d4e17647d033f19f69a3e2e842668eb29863fa459567c9/lib/Text/Megaparsec.dyn_hi
        /home/hasufell/.cabal/store/ghc-9.10.1.20241021/megaparsec-9.6.1-e7391810ad72bf7d47d4e17647d033f19f69a3e2e842668eb29863fa459567c9/lib/Text/Megaparsec.dyn_hi: withBinaryFile: does not exist (No such file or directory)
   |
34 | import Text.Megaparsec
   | ^^^^
```

It was mentioned by @TerrorJack

Reply:
strange, I never needed that. cabal works in mysterious ways :D